src/Elasticsearch.Net/Domain/RequestParameters/RequestParameters.Generated.cs

```diff
@@ -1565,6 +1489,18 @@ public class DeleteByQueryRequestParameters : FluentRequestParameters<DeleteByQu
 		public DeleteByQueryRequestParameters Df(string df) => this.AddQueryString("df", df);
 		
 		
+		///<summary>Specify whether to return detailed information about score computation as part of a hit</summary>
+		public DeleteByQueryRequestParameters Explain(bool explain) => this.AddQueryString("explain", explain);
+		
+		
+		///<summary>A comma-separated list of stored fields to return as part of a hit</summary>
+		public DeleteByQueryRequestParameters StoredFields(params string[] stored_fields) => this.AddQueryString("stored_fields", stored_fields);
```

Comment:
We should probably make sure everything with `*Fields` gets generated as fields that take Fields and Expressions. Like other fields do too. New issue though.